### ISSUE TITLE
continue train problem

### ISSUE THREAD

**Issue by zhaishengfu** [pedrohbtp/snake-rl#1]
hello, I used your code and save and load and save again, but seems each time will learn from scrach. below is my code:
**snake = Snake(width=20)
snake_env = DummyVecEnv([lambda : snake])
policy_kwargs = dict(act_fun=tf.nn.relu, net_arch=[128, 128])
 snake_model = PPO2('MlpPolicy',  snake_env, verbose=0, 
     policy_kwargs=policy_kwargs, 
     learning_rate=LinearSchedule( 40000*5, initial_p=0.0005, final_p=0.00005).value, tensorboard_log="./mlp"
 )**
**snake_model =  PPO2.load("./snake_model6")
snake_model.set_env(snake_env)
for i in range(1000):
     snake_model.learn(total_timesteps=1000, reset_num_timesteps=False)
     end = time.time()
     print('training', i, ' : ', end-start)
     evaluate( snake_model, snake_env, num_steps = 1000)
snake_model.save('./snake_model7')**

snake_model6 is the one that I trained from scrach, and the reward is positive:
**Mean reward: 88.0 Num episodes: 5
training 470  :  326.35501408576965
^[AMean reward: 62.9 Num episodes: 7
training 471  :  327.0486173629761
Mean reward: 44.3 Num episodes: 7
training 472  :  327.73963356018066
Mean reward: 50.0 Num episodes: 1
training 473  :  328.4078848361969
Mean reward: 5.0 Num episodes: 2
training 474  :  329.1125569343567**

but i use above code to load and train again, each time the result seems from scrach:
**training 0  :  1.0612361431121826
Mean reward: -40.5 Num episodes: 19
training 1  :  1.7357032299041748
Mean reward: -35.0 Num episodes: 14
training 2  :  2.3953707218170166
Mean reward: -26.7 Num episodes: 9
training 3  :  3.059976100921631
Mean reward: -29.2 Num episodes: 12
training 4  :  3.736464262008667
Mean reward: -21.8 Num episodes: 11
training 5  :  4.413033723831177
Mean reward: -26.7 Num episodes: 12**

I tested multiple times and each is the same result. What is the problem do you think?

**Comment by pedrohbtp** [comment]
Hello @zhaishengfu, 

I am not sure why loading from disk like you described is not working. 
I saw you also posted the question on the actual stable-baselines library. 
They have more internal knowledge about the library than I do. 
 But I might run the code again later today and try to understand a little more and let you know

**Comment by araffin** [comment]
It might be a bug in the environment: https://github.com/hill-a/stable-baselines/issues/709#issuecomment-592429765
